frontend/src/components/SigninPage.js

```diff
@@ -22,23 +24,22 @@ function SigninPage() {
     const [dialogTitle, setDialogTitle] = useState("")
     const [dialogMsg, setDialogMsg] = useState("")
     const [isSigninSuccess, setIsSigninSuccess] = useState(false)
+    const {user,setUser} = useContext(UserContext)
 
     const handleSignin = async () => {
         setIsSigninSuccess(false)
-        const res = await axios.post(URL_USER_SVC+SIGNIN, { username, password })
+        const res = await axios.post(URL_USER_SVC+SIGNIN, { username, password },{withCredentials:true,credentials: "include"})
             .catch((err) => {
                 if (err.response.status === STATUS_CODE_CONFLICT) {
```

Comment:
Can you change the check for status code conflict to look for 400 instead, cause the login handler never return 409?

Reply:
Changed! Introduced Status code for bad request (Http status 400) in constants folder.